### PR TITLE
[FIX] Helper message Markdown render

### LINKED ISSUE #65: Helper Message Markdown Not Rendering
### Description:

Whenever someone enters the command `/gh help`, the helper message shows up in backticks but the message is not properly rendered inside a markdown block. The message is displayed as plain text.

### Steps to reproduce:

1. Enter `/gh help` in the input message box 
2. Helper Message shows up in plain text.

### Expected behavior:

- Helper message should  be displayed in markdown format as a block of text.

### Actual behavior:
- We get plain text
 - 
image

### PR DESCRIPTION
**Pull request by chinma-yyy**
## Issue(s) 

closes #65 

## Acceptance Criteria fulfillment

- [x] Markdown render of helper message

## Proposed changes (including videos or screenshots)

image

## Further comments

Also I tried to solve the emoji problem but as per my reading i couldnt find a solution. Here is one [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Deprecated_octal) i found out for escaping the octadecimal number which has been deprecated .
The earlier markdown would render a code block like this if corrected :
image

### CODE REVIEW

**Comment by chinma-yyy** [comment]
Should I replace the brackets with a dot which will result in a more organised helper message?
image

**Comment by samad-yar-khan** [comment]
> Should I replace the brackets with a dot which will result in a more organised helper message? image

Hey @chinma-yyy  Sure . Go ahead 💯

**Comment by chinma-yyy** [comment]
> 
> Hey @chinma-yyy Sure . Go ahead 100
@samad-yar-khan Done

**Comment by samad-yar-khan** [comment]
Hey @chinma-yyy I still dont see markdown rendering on it but it does serve the purpose 
image

File: github/lib/helperMessage.ts [hunk 1]
```diff
@@ -22,28 +22,27 @@ export async function helperMessage({
...
+    6. Review a Pull Request -> /github  Username/RepositoryName pulls pullNumber
+    7. Login to GitHub -> /github login
+    8. Logout from GitHub -> /github logout
+    9. View your GitHub Profile and Issues -> /github me
+    10. View/Add/Delete/Update Repository Subscriptions -> /github subscribe
+    11. Subscribe to all repository events -> /github Username/RepositoryName subscribe
+    12. Unsubscribe to all repository events -> /github Username/RepositoryName unsubscribe
+    13. Add New Issues to GitHub Repository -> /github issue
+    14. Search Issues and Pull Requests -> /github search
+    15. Assign and Share Issues -> /github issues
```
Comment (samad-yar-khan) [inline github/lib/helperMessage.ts:39]: @chinma-yyy  try doing doing something like this, it seems hacky but might work to render markdown for the current scenario 
```
${```}
    1. See Interactive Button interface to fetch repository data -> /github Username/RepositoryName
    2. Get details of a Repository -> /github  Username/RepositoryName repo
    3. Get Issues of a Repository -> /github  Username/RepositoryName issues
    4. Get Contributors of a Repository -> /github  Username/RepositoryName contributors
    5. Get Recent Pull Request of a Repository -> /github  Username/RepositoryName pulls
    6. Review a Pull Request -> /github  Username/RepositoryName pulls pullNumber
    7. Login to GitHub -> /github login
    8. Logout from GitHub -> /github logout
    9. View your GitHub Profile and Issues -> /github me
    10. View/Add/Delete/Update Repository Subscriptions -> /github subscribe
    11. Subscribe to all repository events -> /github Username/RepositoryName subscribe
    12. Unsubscribe to all repository events -> /github Username/RepositoryName unsubscribe
    13. Add New Issues to GitHub Repository -> /github issue
    14. Search Issues and Pull Requests -> /github search
    15. Assign and Share Issues -> /github issues
 ${```}
```

**Review by samad-yar-khan — CHANGES_REQUESTED** [review]
The helper message in a markdown block helps us in differentiating it from a normal message in the channel. Trying to render that will definitely improve user experience

File: github/lib/helperMessage.ts (hunk 1 above)
Comment (chinma-yyy) [reply github/lib/helperMessage.ts:39]: image
 Hey @samad-yar-khan Is this fine?

Comment (samad-yar-khan) [reply github/lib/helperMessage.ts:39]: Yeah, that actually seems good. Go ahead @chinma-yyy 🤖

Comment (chinma-yyy) [reply github/lib/helperMessage.ts]: Done @samad-yar-khan :+1:

**Comment by samad-yar-khan** [comment]
LGTM @chinma-yyy . Thanks 🚀

**Review by samad-yar-khan — APPROVED** [review]

**Pull request merged**